administrator/components/com_users/Model/GroupsModel.php

```diff
@@ -221,8 +225,11 @@ private function populateExtraData(array $items)
 		}
 
 		// Get total disabled users in group.
-		$query->clear('where')
-			->where('map.group_id IN (' . implode(',', $groupIds) . ')')
+		$query->clear();
+		$query->select('map.group_id, COUNT(DISTINCT map.user_id) AS user_count')
+			->from($db->quoteName('#__user_usergroup_map', 'map'))
+			->join('LEFT', $db->quoteName('#__users', 'u') . ' ON ' . $db->quoteName('u.id') . ' = ' . $db->quoteName('map.user_id'))
```

Comment:
```suggestion
			->join('LEFT', $db->quoteName('#__users', 'u'), $db->quoteName('u.id') . ' = ' . $db->quoteName('map.user_id'))
```
I think this way will work too as seen elsewhere.

Reply:
probably yes, but as an old sql guy i prefer to mark evidence on sql clause syntax , with the explict `'ON'` (it seems more readable to me) , but that's matter of personal feeling so  .....feel free to have a different opinion .... both ways will work the same ;)

Reply:
I am ok either way. It is more for consistency.